src/Nexus.ts

```diff
@@ -246,6 +280,16 @@ class Nexus {
     }
   }
 
+  private setOAuthCredentials(credentials: types.IOAuthCredentials): void {
```

Comment:
Usually I list the public functions of a class first and the private ones below, so that it's easier to see where the public interface of a class ends.